modules/_labs/panel/react/lib/Panel.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {colors, spacing, CanvasSpacingValue} from '@workday/canvas-kit-react-core';
+
+export interface PanelProps extends React.HTMLAttributes<HTMLDivElement> {
+  padding?: CanvasSpacingValue;
```

Comment:
Can you document these props?

Reply:
yep yep!

Reply:
I meant in the readme haha. However, it sounds like we'll be dropping props in the readme with the new storybook format, so this is probably fine for now.

Reply:
Yeah, I was talking to @NicholasBoll  about this, and since we're moving in this direction, I added them in the file :)